docs/write-speed.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Write speed 
+sidebar_label: Write speed
+slug: /neo4j-slow-write-performance
+---
+
+Fast reads are essential for every database. Once the database gets huge, it needs to perform very fast. But with huge amounts of data in the database, write speed can become an issue. The rising need for better write speed comes from the increased load on every database system and strive for better efficiency. 
+
+## Neo4j slow write performance 
+
+Neo4j database can have an issue with write performance. Take a look at this issue user have reported ["Neo4j outrageous write performance"](https://stackoverflow.com/questions/40870057/neo4j-outrageous-write-performance). If you read an issue, you can notice that the user has tried several approaches to combat slow write performance. First with a bulk of synchronous 1000 queries and transactions, then with one transaction and 1000 queries, and last 1000 async transactions with one query,  all on ten thread machine. All options for 1000 transactions are in the time range of second, which is not great. Some approaches and comments from this issue are great, but let's make a summary that can help you maximize Neo4j write performance. 
+
+## Optimize Neo4j write performance
+
+
+### Check index and unique constraints  
+
+Indexing your data can make a huge difference when searching or traversing the graph. At the same time, any index you create can negatively impact write speed, because it adds additional load to the database engine. The same argument is valid for node labels and unique constraints. Each label and unique constraint is an index. 
+In order to improve your write speed performance, check your indexes, labels, and unique constraints. If you have unnecessary indexes, evaluate whether you can reduce them to improve write speed performance. Of course, there are possible penalties for reading speed performance, but it is a balancing act. 
+
+
+### Baches of transactions
+
+Neo4j store each transaction step in memory. If anything fails, it can roll back all the changes that caused the issue. This is primarily because of being an ACID-compliant database.
```

Comment:
```suggestion
Neo4j stores each transaction step in memory. If anything fails, it can roll back all the changes that caused the issue. This is primarily because it is an ACID-compliant database.
```